keyless/common/src/input_processing/bits.rs

```diff
@@ -1,15 +1,13 @@
 // Copyright © Aptos Foundation
 // SPDX-License-Identifier: Apache-2.0
 
-// Copyright © Aptos Foundation
-
 use anyhow::{bail, Result};
 use std::ops::{self, Add, AddAssign};
 
 /// Type for bit representation during conversion. Represents bits using strings, for easy
 /// manipulation:
 /// ```
-/// use prover_service::input_processing::bits::Bits;
+/// // use prover_service::input_processing::bits::Bits;
```

Comment:
This seems... strange?  Wonder how it compiles / if that's the case you should just delete this line.

Reply:
cc @zjma 
That's a good question. I don't know the intention of this comment.

1. if the comment is used for both testing and showcasing the usage, which is my understanding, I comment out the unnecessary import.
2. if the comment is used for showcasing only, we can add the annotation `ignore` to avoid this code being compiled.
 ref: https://doc.rust-lang.org/rustdoc/write-documentation/documentation-tests.html#attributes

Reply:
i didn't know it can be used for testing and should compile 🤦 

i think i'm just showcasing here.

Reply:
fixed